### PR TITLE
 #167730732 Setup Mocha for unit testing

### PR DESCRIPTION
**Pull request by ola357**
#### What does this PR do?
* Setup unit testing with Mocha js
#### Description of Task to be completed?
* install mocha js
* write hooks and assert values
* configure package.json
* run `npm test`
#### How should this be manually tested?
* git clone this branch `ch-setup-unit-testing-167730732`
* cd to the project
* Checkout to the branch
* run `npm install`
* run `npm test`

#### Any background context you want to provide?
* N/A
#### What are the relevant pivotal tracker stories?
* [#167730732](https://www.pivotaltracker.com/story/show/167730732)
#### Screenshots or gifs (if appropriate)
* N/A
#### Questions:
* N/A

### CODE REVIEW

**Review by marusoft — CHANGES_REQUESTED** [review]
Awesome job @ola357, please provide the link of the branch to be cloned in PR manual test.

**Review by dewaleolaoye — APPROVED** [review]
Good job

**Review by marusoft — APPROVED** [review]

File: test/index.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const assert = require('assert');
```
Comment (rovilay) [inline test/index.js]: Please rename the `test` folder to `__tests__`

Comment (ola357) [reply test/index.js]: Okay, It has been renamed, Please confirm

**Comment by rovilay** [comment]
@ola357 I get this when I run test.
Screenshot 2019-08-19 at 11 09 31 AM

**Comment by rovilay** [comment]
@ola357 Also, Please move the `__tests__` folder inside `src` folder.

File: test/index.js (hunk 1 above)
Comment (rovilay) [reply test/index.js]: Good.

**Comment by rovilay** [comment]
@ola357 Please resolve conflicts

**Comment by rovilay** [comment]
@ola357 Please resolve confilcts.

**Comment by rovilay** [comment]
@ola357 please untrack package-lock.json.

**Pull request merged**